tests/integration/test_charm.py

```diff
@@ -126,10 +136,20 @@ async def test_settings_are_correct(ops_test: OpsTest, unit_id: int):
     # Validate each configuration set by Patroni on PostgreSQL.
     assert settings["archive_command"] == "/bin/true"
     assert settings["archive_mode"] == "on"
+    assert settings["autovacuum"]
```

Comment:
We should most probably still assert the exact value, even if it's just True.

Reply:
Good suggestion. Updated the test on https://github.com/canonical/postgresql-k8s-operator/pull/281/commits/febe5df3e30701377cf1489ba6767bf5397e1470.